homepage/tests.py

```diff
@@ -191,3 +191,111 @@ def test_print_details(capsys, review_id, expected_review_details):
     Review.objects.get(pk=review_id).print_details()
 
     assert capsys.readouterr().out == expected_review_details
+
+# -----------prerequisites tests----------- #
+
+
+# three courses:
+@pytest.fixture
+def courses():
+    courses_list = [
+         Course(1, "Course1", True, 3, "N/A", 1.3, 4, 5, 1),
+         Course(2, "Course2", False, 2, "N/A", 3.5, 2, 13, 5),
+         Course(3, "Course3", False, 1, "N/A", 2, 2.5, 0, 0)]

```

Comment:
It will be more readable when we can easily see the end of the list
```suggestion
         Course(3, "Course3", False, 1, "N/A", 2, 2.5, 0, 0)
   ]
```